eip-2930.md

```diff
@@ -0,0 +1,132 @@
+---
+eip: 2930
+title: Optional access lists
+author: Vitalik Buterin (@vbuterin), Martin Swende (@holiman)
+discussions-to: TBD
```

Comment:
Need this before we merge.

Reply:
Do you think just pointing it to the same place as EIP 2929 discussions is ok? The two EIPs are closely intertwined so talking about them together might be reasonable?